### ISSUE TITLE
Image input for LangGraph integration

### ISSUE THREAD

**Issue by GregorBeuster** [Yonom/assistant-ui#881]
I dug through the code and it seems there is only text content for Langgraph messages?! You DO have some "attachment" components and hooks as it looks like but not working with Langgraph "yet" I guess? Image input would be highly appreciated.

:-) Keep up te great work!

**Comment by Yonom** [comment]
Thanks for the feedback! I can try to get something for langgraph image attachments out by this weekend :)

**Comment by Yonom** [comment]
Hey, yes, the goal is to get to feature parity and have first class support for LangGraph

**Comment by Yonom** [comment]
@GregorBeuster the v0.0.19  of the react-langgraph package adds experimental support for image attachments; simply specify the following flag:

```ts
useLangGraphRuntime({
  unstable_allowImageAttachments: true,
  // rest of your config
```

Specifying this should make an attachment button appear in the composer 

Let me know how it goes :)

**Comment by Yonom** [comment]
ah, thanks for catching this - fixed in v0.0.20

**Comment by Yonom** [comment]
Wonderful! 

Yeah I will be exposing the attachments adapter so you can pass custom adapters as well
